notify.sh

```diff
@@ -11,4 +11,4 @@ curl https://api.rollbar.com/api/1/deploy/ \
   -F access_token=$ROLLBAR_TOKEN \
   -F environment=$ENVIRONMENT \
   -F revision=$CI_COMMIT_ID \
-  -F local_username=$CI_COMMITTER_USERNAME
\ No newline at end of file
+  -F local_username=$CI_COMMITTER_USERNAME
```

Comment:
Did something change here?

Reply:
Added a newline at the end. This is part of the annoyance of not having it be consistent.